api/include/opentelemetry/trace/tracer_provider.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "opentelemetry/nostd/string_view.h"
+#include "opentelemetry/trace/tracer.h"
+
+namespace opentelemetry
+{
+namespace trace
+{
+class TracerProvider
+{
+public:
+  virtual ~TracerProvider()                                                 = default;
+  virtual Tracer *const GetTracer(nostd::string_view library_name,
+                                  nostd::string_view library_version,
```

Comment:
This one is called by the user. Do we want to have a default arg here? `library_version=""`